guides/v2.2/extension-dev-guide/view-models.md

```diff
@@ -0,0 +1,66 @@
+---
+group: php-developer-guide
+subgroup: 99_Module Development
+title: ViewModels
+menu_title: ViewModels
+menu_order: 10001
```

Comment:
Consider adding your name so that the proper attribution shows up in the top right corner of the new topic!

```
contributor_name: Space 48
contributor_link: https://www.space48.com/
```